src/Meshes/interval.jl

```diff
@@ -71,6 +71,20 @@ function boundary_face_name(mesh::IntervalMesh, elem::Integer, face)
     return nothing
 end
 
+function monotonic_check(faces)
+    n = length(faces) - 1
+    monotonic_incr = all(map(i -> faces[i].z < faces[i + 1].z, 1:n))
+    monotonic_decr = all(map(i -> faces[i].z > faces[i + 1].z, 1:n))
+    if !(monotonic_incr || monotonic_decr)
+        error(
+            string(
+                "Faces in vertical mesh are not increasing monotonically. ",
+                "We need to have dz_bottom <= z_max/(nz+1) and dz_top >= z_max/(nz+1).",
```

Comment:
```suggestion
                "We need to have dz_bottom <= z_max/(z_elem+1) and dz_top >= z_max/(z_elem+1).",
```
Could you double check if this is correct (especially the +1 part)?

Reply:
(z_elem is just to match the command line argument. I don't have a strong preference.)

Reply:
I'm not familiar with this relationship, @trontrytel can you confirm?

Reply:
Let me double check with Tapio. I actually think there should be no +1 here.

Reply:
(but, yes, it would be useful if @trontrytel can test!)

Reply:
Ok, I confirmed with Tapio, there should be no +1 here. I updated the suggestion.